clis/teliod/src/main.rs

```diff
@@ -134,5 +139,10 @@ async fn main() -> Result<(), TeliodError> {
                 Err(TeliodError::DaemonIsNotRunning)
             }
         }
+        #[cfg(feature = "qnap")]
+        Cmd::Cgi => {
```

Comment:
So now we have this quite specialized for qnap, so i think it would make sense to also call it a bit more concretely: like telio `qnap-cgi` or leave this as generic cgi, with qnap auth as also a seperate sub part

Reply:
changed